packages/integration-sdk-runtime/src/api/index.ts

```diff
@@ -55,6 +57,10 @@ export function createApiClient({
     retry: retryOptions ?? {},
   };
 
+  if (isLambdaUrl(apiBaseUrl)) {
+    opts.signAwsV4 = { credentials: defaultProvider({ maxRetries: 3 }) };
```

Comment:
It seems to me like we should only load provider credentials once, but given the number of `Could not load any credentials from provider` errors we're seeing, it seems like the `@lifeomic/alpha` package may be re-fetching credentials frequently throughout integration execution. Maybe they're expiring frequently, IDK, but its worth investigating whether the credentials are not properly cached.

Reply:
Actually, yes, it seems that the `awsV4Signature` function is loaded as an axios request interceptor in `@lifeomic/alpha`, which (I think) means that the function is invoked _on each request_. Since a new instance of `defaultCredentials()` is instantiated at https://github.com/LifeOmic/alpha/blob/master/src/interceptors/aws-v4-signature.ts#L68, _each_ lambda call is re-instantiating those credentials - meaning, re-invoking the ECS container metadata endpoint.

It would be better if `@lifeomic/alpha` instantiated `defaultCredentials()` just one time and re-used the same instance in the interceptor.

Reply:
Opened https://github.com/lifeomic/alpha/issues/159 as a recommendation.